devices.js

```diff
@@ -15018,7 +15018,23 @@ const devices = [
             .withSystemMode(['off', 'heat', 'auto'], ea.STATE_SET).withLocalTemperatureCalibration(ea.STATE_SET)
             .withRunningState(['idle', 'heat'], ea.STATE).withAwayMode()],
     },
-
+    // HGKG
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_dzuqwsyg'}],
+        model: 'BAC-002-ALZB',
```

Comment:
When searching for `BAC-002-ALZB` on Google I cannot find anything about this device, where can it be bought?

Reply:
Thank you. I've modified as per your instruction (however I think you meant as below) and removed the converter.
Also added the associated change to its reference in devices.js.

`if (model.model === 'BAC-002-ALZB') {
  return {system_mode: value ? 'cool' : 'off'};
} else {
  return {system_mode: value ? 'heat' : 'off'};
}`

Purchased from Aliexpress. 
https://www.aliexpress.com/item/1005001441524917.html?spm=a2g0s.9042311.0.0.34534c4d0ZdyPu
https://www.aliexpress.com/item/1005001482329022.html?spm=a2g0s.9042311.0.0.27424c4dshq3Ag